apps/blockchain/lib/blockchain/account/inspector.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Blockchain.Account.Inspector do
+  @moduledoc """
+  Module to inspect account information
+  """
+
+  alias Blockchain.Account
+
+  @doc """
+  Prints a visual representation of several accounts
+  """
+  def inspect_accounts(state, account_addresses) do
+    Enum.map(account_addresses, &inspect_account(state, &1))
+  end
+
+  @doc """
+  Prints a visual representation of an account
+  """
+  def inspect_account(state, account_address) do
```

Comment:
will it be used only with a committed state?

Reply:
Yeah. We could improve this as we go. I was just extracting this because I thought it might be helpful as tooling since we're all debugging issues with blocks. I've been using this for what I'm currently debugging.